### ISSUE TITLE
do not remove Sway and i3 from Clear

### ISSUE THREAD

**Issue by paulcarroty** [clearlinux/distribution#2874]
Both are real and useful GNOME alternatives.

https://github.com/clearlinux/clr-bundles/commit/4b52e9f7b8cdf7bd5953fe5eeeae134ae901ceef

**Comment by zearp** [comment]
I have to agree, specially on the Wayland side having a native lightweight window manager is essential for when you don't want or need a full desktop environment. But to be honest I'm surprised they still have desktop iso's on their site. The focus has not been on delivering a desktop experience in a while. Hence many packages simply being removed. Unfortunately. I have stopped using CL as my daily driver desktop about a year ago. The desktop side has not been getting much love and attention from Intel for a while. I wonder what desktop OS the devs use.

CL could've been one of the big disto's (top 5) as it out performs anything else, is stable and secure and has a nice package manager and Intel has enough knowledge and knowhow to provide enterprise support to pay for the developers. Taking some market share from RHEL/Ubuntu would've been good. Competition makes everybody better. And the Linux community in general always benefits from another well maintained desktop OS. But it didn't turn out that way.

tl;dr I hope they reconsider seeing as i3 is a very mature and stable windows manager. The same goes for Sway which is a drop in replacement for i3 on Wayland.

**Comment by nickles95** [comment]
I, too, use sway on CL and was extremely surprised when it stopped working.

**Comment by fenrus75** [comment]
reasonable requests are more likely to be looked at than just
unconstructive yelling.
i was going to look at this tomorrow on the weekend....

**Comment by wheerdam** [comment]
Well it's been a good run using Clearlinux as a desktop distro, but removing i3 will be the last reason I need to switch away.

**Comment by fenrus75** [comment]
is it i3 or sway you really want?
the non-wayland side of the world (xorg) is bit rotting pretty quickly

**Comment by wheerdam** [comment]
I think if Wayland is the sole future, you should keep sway because plenty of power users use it.

For me personally, I still do a lot of screen sharing and screen recording and losing Xorg will be painful in the short term, but I will eventually migrate to Wayland+Pipewire anyway for my work machine.

**Comment by fenrus75** [comment]
sway sounds obvious to get back, I'll poke that tomorrow
i3 I'm more. hesitant about; I'll look but keeping x stuff around is
increasingly painful for us

**Comment by fenrus75** [comment]
fwiw I've added both back; sway is a nobrainer for us but i3 over time we
expect to suffer the same as the rest of the X world (e.g. it's getting
increasingly hard to keep that stuff working since the upstream X world is
basically dead and mesa and others moved on)

if there's applications for sway that package cleanly (read: autospec and
don't need patches) and have an open license I don't mind adding them to
the base OS

**Comment by zearp** [comment]
> > if there's applications for sway that package cleanly (read: autospec and don't need patches) and have an open license I don't mind adding them to the base OS
> 
> foot fuzzel fnott wl-clipboard greetd

None of those is a Sway dependency. 

Recommend to install are, swaybg, swaylock,  swaymsg swayidle and some others I have to look up. There is also xdg wlroots portal and others that come to mind. 

Greetd is not needed. You can use seat or logind as well.

I’m sure the package maintainer knows what Sway actually needs and what not. If not we’re doomed 😉

**Comment by zearp** [comment]
> sway sounds obvious to get back, I'll poke that tomorrow i3 I'm more. hesitant about; I'll look but keeping x stuff around is increasingly painful for us

I completely agree, Wayland is the future. Xorg development is at an all time low. There is little reason to spend time on Xorg, specially when Xwayland exists. I am glad Sway gets to stay! Thank you.

**Comment by greysonofusa** [comment]
Wayland is by far superior. Wayland and Hyprland combined is perfection.
https://github.com/hyprwm/Hyprland Also? any chance we can add this to the official repo for CL? 
sudo swupd bundle-add hyprland

**Comment by paulcarroty** [comment]
Sway bundle is available again, thanks! Closing this now, please create dedicated package request for Hyprland.
